ci-operator/jobs/devfile/devworkspace-operator/devfile-devworkspace-operator-master-presubmits.yaml

```diff
@@ -1,5 +1,52 @@
 presubmits:
   devfile/devworkspace-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
```

Comment:
Is it a branch that is going to be used for image building? Or it's just a marker - against which base branch Job should be triggered?

Reply:
The jobs will run against only master branch